api/proto/teleport/legacy/types/types.proto

```diff
@@ -2918,6 +2918,18 @@ message RoleOptions {
   // database when not set to off.
   // 0 is "unspecified", 1 is "off", 2 is "keep", 3 is "best_effort_drop".
   CreateDatabaseUserMode CreateDatabaseUserMode = 29 [(gogoproto.jsontag) = "create_db_user_mode,omitempty"];
+
+  // MinMFAVerificationInterval defines the minimum duration that must elapse
+  // between successive MFA verifications. This variable is used to ensure
+  // that users are periodically prompted to verify their identity, enhancing
+  // security by preventing prolonged sessions without re-authentication when using
+  // tsh proxy * derivatives.
+  // It's only effective if the session requires MFA.
+  // If not set, defaults to `max_session_ttl`.
+  int64 MinMFAVerificationInterval = 30 [
```

Comment:
Now that Alan imported the gogo proto wrappers, can we use the `google.protobuf.Duration` instead of the typecast?

Reply:
I tried it and it didn't play well, we get a `Duration` struct that is not nice to use, not the `durationpb` one. So I think better keep the legacy approach with the typecast

Reply:
Did you include de std duration flag for gogo? It generates a time.Duration

Reply:
Yeah, I missed that. Now looks ok, changed https://github.com/gravitational/teleport/pull/45569/commits/1fdb62b16eec39e3f8c56dfa26cfce6236fe0fa2